src/einsteinpy/tests/test_coordinates/test_coord_transform.py

```diff
@@ -104,6 +104,32 @@ def test_cycle_BLSpherical(bl):
 # Tests for object.__repr__ and object.__str__
```

Comment:
Please add the new tests above this line. This line was meant to demarcate that tests for `__str__` and `__repr__` are written below here.